examples/stable-diffusion-inpainting/examples.yaml

```diff
@@ -0,0 +1,5 @@
+- input:
```

Comment:
How big are these images, maybe put them in git-lfs if they are more than a few kbs.

Reply:
the images are encoded as base64 in the examples.yaml files. the two images in the inpainting example are about 400KB and 10KB and the image in the img2img example is about 400KB

Reply:
its 512x512

Reply:
I think that's ok.

Reply:
cool. I'll merge then